src/db/schema.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  sqliteTable,
+  integer,
+  text,
+  foreignKey
+} from 'drizzle-orm/sqlite-core';
+
+// ---------------------------------------
+// TEACHER
+// ---------------------------------------
+export const teacher = sqliteTable('teacher', {
+  id: integer('id', { mode: 'number' }).primaryKey({ autoIncrement: true }),
+  first_name: text('first_name').notNull(),  // First name
+  last_name: text('last_name').notNull(),    // Last name
+  email: text('email').notNull(),            // Email
+  phone_number: text('phone_number'),        // Phone number
+  avatar: text('avatar'),                    // Avatar
```

Comment:
Hi, Avatar brauchen wir nicht als Attribut. Ich habe ein Component in der Lehrer-Ansicht, die ein Avatar aus den ersten Buchstaben des Namens automatisch bildet. Danke trotzdem!

Reply:
Alles klar, werde ich entfernen!